applications/GeoMechanicsApplication/tests/cpp_tests/test_t_normal_flux_condition.cpp

```diff
@@ -0,0 +1,314 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#include "containers/model.h"
+#include "custom_conditions/T_normal_flux_condition.h"
+#include "geo_mechanics_application_variables.h"
+#include "includes/condition.h"
+#include "testing/testing.h"
+
+using namespace Kratos;
+
+namespace Kratos::Testing {
+
+void TestTnormalFluxCondition(ModelPart& rModelPart, std::vector<double> RHS)
```

Comment:
Minor efficiency thing and suggested name:
```suggestion
void TestTnormalFluxCondition(ModelPart& rModelPart, const std::vector<double>& rExpectedRightHandSide)
```

Reply:
Richard, thank you. It is much better.